### DISCUSSION TITLE
Cannot update to version

### DISCUSSION THREAD

**Discussion by yousefvand** [evilsocket/opensnitch#590] [Releases]
On Archlinux, using `yay`, installing `opensnitch-git` package, I cannot update from `1.5.0.r6.9770cf5-1` to `1.5.0.r29.d264bb0-2`

I get this error:

```
cannot find package "github.com/golang/protobuf/protoc-gen-go" in any of:
        /usr/lib/go/src/github.com/golang/protobuf/protoc-gen-go (from $GOROOT)
        /home/me/.cache/yay/opensnitch-git/src/gopath/src/github.com/golang/protobuf/protoc-gen-go (from $GOPATH)
==> ERROR: A failure occurred in build().
    Aborting...
 -> error making: opensnitch-git
```

I don't set `GOPATH` so by defalt it is `~/go`. If I run `go get -u github.com/golang/protobuf/protoc-gen-go` I get `protoc-gen-go` in `~/go/bin` but still update fails as described.

**Comment by gustavo-iniguez-goya** [comment]
Hi @yousefvand ,

Did you try asking on the Arch forums?
